modules/PhyVersoBSP/phyverso_mcu_comms/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.10)
+
+# set the project name
+project(phyverso_mcu_comms VERSION 0.1)
+# specify the C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
this could be replaced by 
`target_compile_features(phyverso_mcu_comms PRIVATE cxx_std_17)`
after the target has been defined further down